drivers/nfs.py

```diff
@@ -50,34 +51,41 @@
 NFS_SERVICE_WAIT = 30
 NFS_SERVICE_RETRY = 6
 
+NFS4_PSEUDOFS = "/"
+NFS4_TMP_MOUNTPOINT = "/tmp/mnt"
 
 class NfsException(Exception):
 
     def __init__(self, errstr):
         self.errstr = errstr
 
-
-def check_server_tcp(server, nfsversion=DEFAULT_NFSVERSION):
+def check_server_tcp(server, transport, nfsversion=DEFAULT_NFSVERSION):
```

Comment:
The method is explicit (or is expected to be) in using TCP, so it should not have the transport argument. If `get_suppoted_nfs_versions` needs a transport it should be passed as a literal from this method without changing the signature of the method.

Reply:
Indeed but transport can be `tcp` or `tcp6` in the case of IPv6.

Reply:
Ah, I see, that was not clear. Might have been better for the IPv6 bits to be at least a commit on their own right to make it clear.